src/mem/ruby/system/RubySystem.cc

```diff
@@ -493,9 +504,17 @@ RubySystem::resetStats()
     ClockedObject::resetStats();
 }
 
-#ifndef PARTIAL_FUNC_READS
```

Comment:
Nice, this was always a source of confusion which on is being used at runtime. Should be much more clear now!